server/graphql/v2/input/PaymentMethodInput.ts

```diff
@@ -84,7 +84,7 @@ export const getLegacyPaymentMethodFromPaymentMethodInput = async (
     };
     if (pm.creditCardInfo.expYear && pm.creditCardInfo.expMonth) {
       const { expYear, expMonth } = pm.creditCardInfo;
-      paymentMethod['expiryDate'] = moment.utc(`${expYear}-${expMonth}`, 'YYYY-MM');
+      paymentMethod['expiryDate'] = moment.utc(`${expYear}-${expMonth}`, 'YYYY-MM').endOf('month');
```

Comment:
What about the one already added since the PR was deployed?

Reply:
It's covered by the migration?

Reply:
Yes, the migration will update existing credit cards.